app/routes/grants/index.js

```diff
@@ -31,13 +31,17 @@ export default Route.extend({
         // 'projectName'
       ],
       query: {
-        bool: {
-          should: [
-            { term: { pi: user.get('id') } },
-            { term: { coPis: user.get('id') } }
-          ],
+        constant_score : {
           filter: {
-            range: { "endDate": { "gte": "2011-01-01" }} 
+            bool: {
+              should: [
+                { term: { pi: user.get('id') } },
+                { term: { coPis: user.get('id') } }
+              ],
+              must : {
+                range: { "endDate": { "gte": "2011-01-01" } }
```

Comment:
> `range: { endDate: { gte: '2011-01-01' } }`

* Keys should be unquoted unless they contain illegal characters
* Single quotes should be used for string values